ddtrace/llmobs/_evaluators/sampler.py

```diff
@@ -57,8 +59,15 @@ def sample(self, evaluator_label, span):
     def parse_rules(self) -> List[EvaluatorRunnerSamplingRule]:
         rules = []
         sampling_rules_str = os.getenv(self.SAMPLING_RULES_ENV_VAR)
+        telemetry_writer.add_configuration("_DD_LLMOBS_EVALUATOR_SAMPLING_RULES", sampling_rules_str, origin="env")
 
         def parsing_failed_because(msg, maybe_throw_this):
+            telemetry_writer.add_log(
+                TELEMETRY_LOG_LEVEL.ERROR, message="Evaluator sampling parsing failure because: {}".format(msg)
+            )
+            telemetry_writer.add_count_metric(
+                namespace="llmobs", name="evaluators.sampling_rule_parsing_failure", value=1
```

Comment:
might be worth simplifying this to just `evaluators.error` with a `reason:sampling_rule_parsing_failure` tag so we can use the metric for other types of errors as well